hotel/views/hotel_view.xml

```diff
@@ -492,6 +492,7 @@
                      <button name="action_cancel_draft" states="cancel" string="Set to Draft" type="object" icon="gtk-file" class="oe_highlight"/>
                      <button name="action_cancel" string="Cancel Folio" states="draft,manual,progress" type="object" icon="gtk-cancel" class="oe_highlight"/>
                      <button name="invoice_cancel" string="Cancel Folio" states="invoice_except" icon="gtk-cancel" class="oe_highlight"/>
+                     <button name="action_done" string="Set to Done" states="progress" type="object" icon="gtk-apply" class="oe_highlight"/>
```

Comment:
manual +progress na hovu joie?

Reply:
And Set to draft ma cancel +  done?

Reply:
When folio is done state then all process are done so not need to set as a draft state.